arangosh/Shell/V8ShellFeature.cpp

```diff
@@ -511,6 +511,31 @@ int V8ShellFeature::runShell(std::vector<std::string> const& positionals) {
       input = "help()";
     }
 
+    if (i[0] == 'a') {
```

Comment:
Undefined behavior if `i` is empty.
```suggestion
    if (i[0] == 'a') {
```